lib/services/pages.test.js

```diff
@@ -10,25 +10,26 @@ const _ = require('lodash'),
   notifications = require('./notifications'),
   sinon = require('sinon'),
   siteService = require('./sites'),
-  timer = require('../timer'),
-  winston = require('winston');
+  timer = require('../timer');
```

Comment:
does winston need to stay for backwards compatibility?

Reply:
Yes